python/sparkdl/transformers/tf_tensor.py

```diff
@@ -66,65 +63,29 @@ def setParams(self, tfInputGraph=None, inputMapping=None, outputMapping=None, tf
         # Further conanonicalization, e.g. converting dict to sorted str pairs happens here
         return self._set(**kwargs)
 
-    def _getSparkDlOpName(self, tensor_name):
-        """
-        Given a tensor name, returns the name of the op generating the tensor, prefixed with
-        a special scope indicating that the op has been added by Sparkdl.
-        """
-        op_name = tfx.op_name(tensor_name)
-        return tfx.add_scope_to_name(scope=self.SPARKDL_OP_SCOPE, name=op_name)
-
-    def _addCastOps(self, user_graph_def):
```

Comment:
Remove logic for injecting casting ops before input tensors to convert from float64 -> expected data type